src/PasswordField.js

```diff
@@ -20,15 +20,12 @@ const getStyles = (props, context, state) => {
 
   const styles = {
     root: {
-      display: 'flex',
-      width: props.fullWidth ? '100%' : null
-    },
-    inputContainer: {
-      flex: 1
+      position: 'relative',
```

Comment:
Are there any ceveats when setting this component to `position: relative`? I.e. any unintended side-effects this may cause when using this component?
I'm asking because this affects the layouting and the component gets a different `zIndex` and so on… Not sure about this. :confused: